src/main/java/service/impl/ParseServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package service.impl;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import model.FruitTransaction;
+import service.ParseService;
+
+public class ParseServiceImpl implements ParseService {
+    public static final String SPLITTER_COMMA = ",";
+    public static final int TYPE_INDEX = 0;
+    public static final int FRUIT_INDEX = 1;
+    public static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> parseFruitTransaction(List<String> records) {
+        return records.stream()
+                .skip(0)
+                .map(this::getFromCsvRow)
+                .collect(Collectors.toList());
+    }
+
+    private FruitTransaction getFromCsvRow(String line) {
+        String[] fields = line.split(SPLITTER_COMMA);
```

Comment:
Consider using a more descriptive constant name for the comma, such as 'CSV_DELIMITER', to improve code readability.